annotation/src/main/java/org/cbioportal/genome_nexus/annotation/web/AnnotationController.java

```diff
@@ -382,6 +382,7 @@ private VariantAnnotation getVariantAnnotation(String variant)
                     // this is thrown when the annotationJSON can't be stored by mongo
                     // due to the variant annotation key being too large to index
                     LOG.info(e.getLocalizedMessage());
+                    return variantAnnotation;
```

Comment:
I agree with Rob on this. I have an instance of genome nexus running master here: http://dashi-dev.cbio.mskcc.org:5556. I tried the links you provided in slack and it returns the correct annotation.